scipy/ndimage/tests/test_filters.py

```diff
@@ -2008,9 +2008,9 @@ def test_rank15(self, dtype, xp):
 
     def test_rank16(self, xp):
         # test that lists are accepted and interpreted as numpy arrays
-        array = [3, 2, 5, 1, 4]
+        array = xp.asarray([3, 2, 5, 1, 4])
```

Comment:
This test is meant be test array-likes are accepted and converted to numpy. Maybe it should be decorated by `@skip_xp_backends(np_only=True)` for viz.